src/index.html

```diff
@@ -6,9 +6,28 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#"><img src="./images/logo.png" alt="logo Moyo"></a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li><a href="#" class="is-active">Apple</a></li>
+          <li><a href="#">Samsung</a></li>
+          <li><a href="#">Smartphones</a></li>
+          <li><a href="#" data-qa="hover">Laptops & Computers</a></li>
+          <li><a href="#">Gadgets</a></li>
+          <li><a href="#">Tablets</a></li>
+          <li><a href="#">Photo</a></li>
+          <li><a href="#">Video</a></li>
+        </ul>
+      </nav>
+    </header>
+    <!-- <h1>Moyo header</h1> -->
```

Comment:
Remove this comment, it does not make any sense to the person who reviews your work or tasks, in the future you should leave only those comments on your work that will help another developer understand how your code works.